examples/stable_diffusion_v2/ldm/modules/train/optim.py

```diff
@@ -56,6 +56,33 @@ def decay_filter(param):
 
         return all([x not in param.name.lower() for x in filter_list])
 
+    def scale_lr(group_params, lr):
```

Comment:
This `scale_lr` is a function that only applies for `controlnet`. I suggest to pass `scale_lr` as an argument instead of a default operation inside `build_optimizer`